public/locales/de/translation.json

```
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "In diesem Spiel gibt es zwei Rollen: die investigierende und die lügende Person. Jede Runde wechselt sich diese Rolle.",
+      "Die lügenden Person werden zwei zufällige Wikipedia Artikel präsentiert. Nur einer der beiden Artikel kann gelesen werden.",
+      "Die investigierende Person sieht die Titel beider Artikel und bekommt die möglichkeit, fragen zu den Artikeln zu stellen.",
+      "Wird die investigierende Person herausfinden, welchen Artikel sich die lügende Person ausgedacht hat?"
+    ],
+    "language": "<0>sprache: </0>",
+    "player-one": "<0>spieler eins: </0>",
+    "player-two": "<0>spieler zwei: </0>",
+    "player-identical": "Spieler können nicht den gleichen Namen haben",
+    "start": "Start!",
+    "footer": "Dieses Spiel wurde inspiriert von <2>Two Of These People Are Lying</2> von Tom Scott und Matt Grey."
+  },
+  "game": {
+    "try-again": "try again",
```

```suggestion
    "try-again": "versuchen Sie es nochmal",
```

or 

```suggestion
    "try-again": "versuche nochmal",
```

(not sure which fits best)